src/Image.h

```diff
@@ -26,8 +26,13 @@ class Image {
 
  public:
   explicit Image(std::size_t capacity)
-      : m_data{capacity, default_init_allocator<char>{}} {
+#ifdef _WIN32
+      : m_data() {
```

Comment:
Don't need the empty m_data constructor.  This might be cleaner just with an #ifdef around the whole constructor.